app/src/main/java/de/tum/in/tumcampusapp/component/ui/news/KinoDetailsFragment.java

```diff
@@ -169,9 +171,14 @@ public void onError(Exception e) {
 
         rootView.addView(headerView);
     }
-
+    //open show_ticket activity, and transfer current movie data to it.
+    //The String data is just a example, it should be get from correspongding movie which now show on the screen.
+    //time and place of movie  is fixed. Only the movie title and date should be transfered to show_ticket activity
     private void showTicket(){
-        // TODO: go to QR code activity (to be implemented)
+        String data = "KingsMan 08.05 " + "\n"+ "Filmbegin: 20:00 o'clock "+"\n"+ " 1. Stock, Hörsaal 1200 (Carl-von-Linde-Hörsaal) Arcisstraße 21";
+        Intent intent = new Intent(getActivity().getApplicationContext(), show_ticket.class);
+        intent.putExtra("movie_data", data);
```

Comment:
Why are you not using a model klass here? (Kotlin data class)

Reply:
It is the work for Michael in #813 . I should add //TODO: here. I will fix that.
ps: Here I just want to transfer some mockup data to my show_ticket activity